test/truffle/compiler/graphs/run

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+
+NATIVE = `tool/jt.rb ruby --version` =~ /Native/
```

Comment:
Use `TruffleRuby.native?` and remove the extra constant

Reply:
Is TruffleRuby put on the path by JT?

Reply:
I see, this file is just run by `ruby` in PATH, that's an anti-pattern, it should use either truffleruby (via `jt ruby`) or `$SYSTEM_RUBY`.

Reply:
This is unused now